arwen/elrondapi/bigintOps.go

```diff
@@ -289,6 +289,11 @@ func bigIntStorageStoreUnsigned(context unsafe.Pointer, keyOffset int32, keyLeng
 		return 0
 	}
 
+	if isElrondReservedKey(key) {
```

Comment:
This verification is best done in the `storageContext.SetStorage`, for all storage writes, because it's a universal restriction.

Reply:
+1

Reply:
+1